### PR TITLE
#340 Continued implementation of forgot_password

### PR DESCRIPTION
**Pull request by marceloamadeu**
For #340 :
Continuing the forgot_password implementation:

- Fixed in view a forgot_password().
- Started forgot_password(email) in auth.py. Now needing finalization.
- Included test_forgot_password_post.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#359](https://github.com/timelesslounge/timelessis/issues/359) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#359](https://github.com/timelesslounge/timelessis/issues/359) is assigned to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/ddbc628e-0b93-4ec4-81ae-47450e63599e); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: timeless/templates/auth/forgot_password.html [hunk 1]
```diff
@@ -15,4 +15,4 @@ <h1>Forgot your password?</h1>
     <input type="submit" value="Forgot my password">
   </form>
 {% endblock %}
-{% endcache %}
\ No newline at end of file
+{% endcache %}
```
Comment (ammaratef45) [inline timeless/templates/auth/forgot_password.html]: @marceloamadeu Do we need to cache forgot password page?

File: tests/integration/it_auth_test.py [hunk 2]
```diff
@@ -57,3 +57,12 @@ def test_forgot_password(client):
 
 def test_activate(client):
     assert client.get("/auth/activate").status_code == 405
+
+
+def test_forgot_password_post(client):
```
Comment (ammaratef45) [inline tests/integration/it_auth_test.py]: @marceloamadeu The test fails

**Comment by ammaratef45** [comment]
@marceloamadeu I have added two comments. Please check them.

File: timeless/templates/auth/forgot_password.html (hunk 1 above)
Comment (marceloamadeu) [reply timeless/templates/auth/forgot_password.html]: @ammaratef45 Truth. I also believe you do not have to. Retired.

File: tests/integration/it_auth_test.py (hunk 2 above)
Comment (marceloamadeu) [reply tests/integration/it_auth_test.py]: @ammaratef45 Adjusted.

Comment (marceloamadeu) [reply tests/integration/it_auth_test.py]: @ammaratef45 Sorted out. I'm sorry for the large number of commits. I was having trouble with my machine's pytest. Now it's settled, too.

**Comment by marceloamadeu** [comment]
@ammaratef45 ping

**Comment by ammaratef45** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 19 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @ammaratef45[/z](https://www.0crat.com/u/ammaratef45), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#359](https://github.com/timelesslounge/timelessis/issues/359) is now out of scope

**Comment by ypshenychka** [comment]
@ammaratef45 According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ammaratef45** [comment]
@ypshebychka confirm

**Comment by ypshenychka** [comment]
@ammaratef45 thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**